### PR TITLE
add missing build command

### PR DESCRIPTION
**Pull request by bertux**
without building first it can't find the dist directory from where it tries to start

### CODE REVIEW

File: apps/ensemble-engine/src/main.ts
Comment (leonprou) [inline apps/ensemble-engine/src/main.ts]: Need to move those prints to the services where they are used/loaded. This way it's messy

**Review by leonprou — CHANGES_REQUESTED** [review]
Thanks for the fixes!
Please see my comment here - https://github.com/ensemble-codes/ensemble-engine/pull/4/files#r1764865122

File: apps/ensemble-engine/src/main.ts
Comment (bertux) [reply apps/ensemble-engine/src/main.ts]: So let's just remove them first.
It's not needed to make it work and it's already warned for RPC when missing and printed when it's not missing.
So another PR should be done to manage the MongoDB URI the same way than RPC URL.

**Comment by bertux** [comment]
> Thanks for the fixes! Please see my comment here - https://github.com/ensemble-codes/ensemble-engine/pull/4/files#r1764865122

I agree with you, I added the logging where the dotconfig was imported but it's only needed for the debug logging so it's better to remove it, that's what I did.
Now it's waiting for your final review and merge.

**Review by leonprou — APPROVED** [review]
LGTM!

**Pull request merged**
